website/src/main/tut/docs/rules/DisableUnless.md

```diff
@@ -14,26 +14,36 @@ within the given "safe" blocks are banned again, to avoid leakage.
 
 ## Configuration
 
-By default, this rule does allows all symbols. To disallow a symbol in a block:
+By default, this rule does allows all symbols. To disallow symbols in a block:
 ```
 DisableUnless.symbols = [
   {
-    block = "scala.Option"
-    symbol = "dangerousFunction"
-    message = "the function may return null"
+      unless = "scala.Option"
```

Comment:
I have a feeling will eventually want to support a list of symbols here, but we can handle that case once the need arise without breaking existing functionality.